awscli/clidriver.py

```diff
@@ -598,6 +610,9 @@ def _create_argument_table(self):
         self._emit('building-argument-table.%s.%s' % (self._parent_name,
                                                       self._name),
                    operation=self._operation_object,
+                   operation_model=self._operation_model,
+                   session=self._session,
+                   command=self,
```

Comment:
I think I am in favor of adding the command object to the event. But could we name it ``command_object`` to be consistent with ``building-command-table`` event of both the data driven and custom commands.